website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+|![Diagram depicting how FLINT calls modules to move carbon between pools.](../static/img/operations-module-case-study.png) |
+|:--:|
+| Diagram depicting how FLINT calls modules to move carbon between pools. |
+
+### Modules
+
+Modules are the building blocks of FLINT. They contain the operations that describe the ecological processes driving carbon changes in the landscape. This case study provides information regarding the major modules present in FLINT.
+
+### Tiers of module
+
+All the modules are divided into three different tiers according to the amount of information required, and the degree of analytical complexity. Three tiers are described for categorizing both emissions factors and activity data. Tier 1 is the basic method, frequently utilizing IPCC-recommended country-level defaults. While Tiers 2 and 3 are each more demanding in terms of complexity and data requirements.
+
+### Temporal distribution
+
+The temporal scale in FLINT is referred to as time-steps. Time-steps are lengths of time over which operations are reported. It is only at the end of a time-step that carbon can be moved from one pool to another.
+
+The standard time-step in FLINT is one month. However it can be varied by the user. One month is the recommended time-step for modelling carbon.
+
+### Simulation units
+
+A Simulation Unit is a unit for which a module is applied. A Simulation Unit can represent a spatial area, such as a pixel or forest stand, or it can represent an emissions source, such as livestock. Here the Simulation Unit refers to a geographically referenced area, it is known as a Land Unit.
+
+The overall framework of FLINT manages the processing of Simulation Units over time. Simulation Units are the basis of all simulations run in FLINT.
+
+### Synchronized events
+
+In some circumstances, modules that are simulated for a particular Simulation Unit are dependent on other Simulation Units. That is, there is an interdependency between the Simulation Units. In these situations it is necessary for there to be a process that synchronizes across the related Simulation Units and this is known as synchronized events. The Synchronized Server in FLINT manages Synchronized events.
+
+### Mass balance
+
+The carbon cycle is based on the transfer of carbon molecules from one pool to another, where the molecules are not created, nor are they destroyed. As such, all outputs must be equal to the inputs. The system must be in balance. This is referred to as mass balance. Overall this is the high level design of FLINT.
+
+|![Diagram depicting the high-level design of FLINT.](../static/img/flint-design-module-case-study.png) |
+|:--:|
+| Diagram depicting the high-level design of FLINT. |
```

Comment:
This describes the system architecture in more detail than is probably necessary. For example, modules are only one small box, where the others are related to external dependencies and input data types. I would try to think of a simpler diagram to explain modules.